draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3084,49 +3090,86 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
+		along with CI/T interface versions supported by these endpoints. 
+	   	Capability type is "FCI.CITEndpoints".
+		For simplicity, it is recommended to minimize the set of supported versions declared by the dCDN. 
+		For example, one way to minimize supported versions is to support only objects of version 2.
+           	Version 1, as originally defined in <xref target="RFC8007"/>, is the default if this capability is not explicitly declared.
         </t>
 
-        <t><list style="empty">
-           <t>Property: versions<list style="empty">
+	<t>
+        A CI/T Trigger Endpoints capability object is encoded as an array of JSON objects containing the following
+        name/value pairs:
+	</t>
+
+        <t><list style="empty" hangIndent="3">
+	<t><list style="hanging" hangIndent="3">
 
-                 <t>Description: A list of version numbers.</t>
+		<t hangText="Name: trigger-endpoint-uri">
+		<vspace blankLines="1"/>
+		Description: Trigger endpoint URI 
+		<vspace blankLines="1"/>
+		Value: A URL represented as a JSON string.
+		<vspace blankLines="1"/>
+		Mandatory: Yes.
+		</t>
 
-                 <t>Type: An array of JSON strings</t>
+        	<t hangText="Name: trigger-versions">
+		<vspace blankLines="1"/>
+		Description: A list of CI/T versions supported by the trigger endpoint.
+		<vspace blankLines="1"/>
+		Value: An array of JSON strings.
+		<xref target="loop-detection-and-prevention"/>.
+		<vspace blankLines="1"/>
+		Mandatory: No. The default is version 2. 
+		A missing or an empty versions list means that only version 2 of the interface and objects is supported.
+		</t>
+
+		<t hangText="Name: trigger-subject">
+		<vspace blankLines="1"/>
+		Description: Array of trigger subjects supported by the trigger endpoint .
+		<vspace blankLines="1"/>
+		Value: An array of Strings containing the type of the subject matching the generic-trigger-spec-value property, 
+		such as "content" or "metadata" as defined in <xref target="trigger-subject"/>.
+		<vspace blankLines="1"/>
+		Mandatory: No. A missing or an empty trigger subjects lists means that all trigger subjects 
+		are supported by the endpoint.
+		</t>
 
-                 <t>Mandatory: No. The default is version 1. A missing or an empty versions list
-                    means that only version 1 of the interface and objects is supported.</t>
-           </list></t>
         </list></t>
-        <section anchor="versions-capability-object" title="CI/T Objects Versions Capability Object Serialization">
+        </list></t>
+
+        <section anchor="cit-endpoints-capability-object-serialization" title="CI/T Endpoints Capability Object Serialization">
           <t>
-             The following shows an example of CI/T Objects Versions Capability object serialization for a dCDN that supports
+             The following shows an example of CI/T Endpoints Capability object serialization for a dCDN that supports
```

Comment:
Here, we just called it CI/T Endpoints, as I recommended above.  We can use either one, but we should be consistent.  I kind of prefer just "Endpoints" as mentioned earlier, but could be persuaded either way if you feel strongly about it.

Reply:
Agreed, the previous one was an omission